main.c

```diff
@@ -67,26 +104,38 @@ int main(int argc, char *argv[], char *envp[])
 		{ "root",       required_argument,  NULL,           'r' },
 
 		/* long options without shorthand */
-		{ "workdir",    required_argument,  NULL,           OPTION_WORKDIR  },
-		{ "mount",      required_argument,  NULL,           OPTION_MOUNT    },
-		{ "mutable",    required_argument,  NULL,           OPTION_MUTABLE  },
-		{ "uid",        required_argument,  NULL,           OPTION_UID      },
-		{ "gid",        required_argument,  NULL,           OPTION_GID      },
-		{ "groups",     required_argument,  NULL,           OPTION_GROUPS   },
-		{ "arch",       required_argument,  NULL,           OPTION_ARCH     },
-		{ "share",      required_argument,  NULL,           OPTION_SHARE    },
-		{ "argv0",      required_argument,  NULL,           OPTION_ARGV0    },
-		{ "hostname",   required_argument,  NULL,           OPTION_HOSTNAME },
-		{ "domainname", required_argument,  NULL,           OPTION_DOMAIN   },
-		{ "time",       required_argument,  NULL,           OPTION_TIME     },
+		{ "workdir",      required_argument, NULL, OPTION_WORKDIR      },
```

Comment:
Micro-nit: I typically align the required/optional/no_argument field on a 8-char width boundary, that leaves us with a bit of breathing space when even longer options are introduced, as we don't have to reformat everything then.

Reply:
an 8-char boundary, LOL.  I was wondering how you picked the column.